tests/NClient.Providers/NClient.Providers.Mapping.HttpResponses.Tests/HttpResponseDeconstructTest.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using AutoFixture;
+using AutoFixture.AutoMoq;
+using FluentAssertions;
+using NClient.Providers.Results.HttpResults;
+using NClient.Providers.Transport;
+using NUnit.Framework;
+
+namespace NClient.Providers.Mapping.HttpResponses.Tests
+{
+    [Parallelizable]
+    public class HttpResponseDeconstructTest
```

Comment:
The name of the class does not match the content of the tests, because there are tests for `Response*` here.
I suggest making a separate class for `Respons*`.

Reply:
I've separated these tests